kratos/input_output/vtk_output.cpp

```diff
@@ -197,96 +197,94 @@ VtkOutput::WriteDataType VtkOutput::GetDataCharacterstic(const std::string& rVar
     else if (KratosComponents<Variable<array_1d<double, 9>>>::Has(rVariableName))
         return  VtkOutput::WriteDataType::VTK_VECTOR_9;
     else
-        KRATOS_ERROR << "Variable: \"" << rVariableName << "\" does not exist." << std::endl;
+        KRATOS_ERROR << "Variable: \"" << rVariableName << "\" not suitable "
```

Comment:
Does this mean that a variable can exist outside KratosComponents ?

Reply:
this class only supports certain variable-types
Other variable-types can exist (and do exist e.g. `ConstitutiveLaw`) but cannot be supported here
hence the error message was wrong

Reply:
Then instead  of `not suitable` shouldn't it be `not supported` ?